### ISSUE TITLE
Fix ComputationalGridGenerator

### ISSUE THREAD

**Issue by andy9t7** [SlicerCBM/SlicerCBM#50]
- [x] Remove unused gmsh_api:
https://github.com/SlicerCBM/SlicerCBM/blob/bddc6b10dfd01158d33cd50fec74d9ba0d840441/ComputationalGridGenerator/ComputationalGridGenerator/ComputationalGridGenerator.py#L7-L8

- [x] Mesh 3D not 2D by uncommenting some code as described in the comment below:
https://github.com/SlicerCBM/SlicerCBM/blob/bddc6b10dfd01158d33cd50fec74d9ba0d840441/ComputationalGridGenerator/ComputationalGridGenerator/ComputationalGridGenerator.py#L519

- [x] Change "Input Folder" to "Output Folder" and move to "Outputs" in GUI.

- [x] Remove unused invert threshold option.

- [x] Check if "Wrap Solidify" effect has been installed before running this code, and prompt user to install SurfaceWrapSolidify extension:
  ```
      #use shrink wrapping to wrap the bone and get the full surface
      segmentEditorWidget.setActiveEffectByName("Wrap Solidify")
      effect = segmentEditorWidget.activeEffect()
      segmentEditorWidget.setCurrentSegmentID(segmentationNode.GetSegmentation().GetSegmentIdBySegmentName("cran"))
      effect.setParameter("region", "outerSurface")
  ```
  to avoid this error:
  ```
  Traceback (most recent call last):
    File "/home/ben/projects/SlicerCBM/SlicerCBM/ComputationalGridGenerator/ComputationalGridGenerator/ComputationalGridGenerator.py", line 273, in onApplyButton
      self.logic.process(self.ui.inputSelector.currentNode(), self.ui.fold_save.currentPath)
    File "/home/ben/projects/SlicerCBM/SlicerCBM/ComputationalGridGenerator/ComputationalGridGenerator/ComputationalGridGenerator.py", line 365, in process
      effect.setParameter("region", "outerSurface")
  AttributeError: 'NoneType' object has no attribute 'setParameter'
  ```

- [x] ~~Add SurfaceWrapSolidify extension as a dependency CMakeLists.txt~~ Remove SurfaceWrapSolidify effect instead. It is not needed if thresholding is done correctly.

- [x] Fix error: "Failed to compute results: File (/home/ben/scratch/ComputationalGridGenerator/Segmentation_1_cran.stl) not found"

**Comment by benzwick** [comment]
Saima says code after line 519 should be uncommented instead of changing 2 to 3

**Comment by benzwick** [comment]
See #57

**Comment by benzwick** [comment]
Fixed in #57
